executor/builder.go

```diff
@@ -86,7 +86,8 @@ var (
 type executorBuilder struct {
 	ctx              sessionctx.Context
 	is               infoschema.InfoSchema
-	snapshotTS       uint64 // The consistent snapshot timestamp for the executor to read data.
+	snapshotTS       uint64 // The ts for snapshot-read, if a select statement without for update will use this ts
+	minForUpdateTS   uint64 // The min ts should be used by insert/update/delete/select-for-update statement
```

Comment:
> Btw what does `min` mean here?

For example, when buildUpdate, `b. minForUpdateTS = b.ctx.GetSessionVars().TxnCtx.GetForUpdateTS()` is set. But when the inner plan is `PointGet` , the builder will skip fetch the latest ts so the it will return 0 ... It is the old logical and I don't want to change it in this PR. So the actual meaning of this variable is "updateTS should not smaller than it" instead of "updateTS should be it"